src/dnsmasq/cache.c

```diff
@@ -1687,14 +1683,6 @@ char *record_source(unsigned int index)
     return "config";
   else if (index == SRC_HOSTS)
     return HOSTSFILE;
-  /*----- Pi-hole modification -----*/
-  else if (index == SRC_REGEX)
-    return SRC_REGEX_NAME;
-  else if (index == SRC_GRAVITY)
-    return SRC_GRAVITY_NAME;
-  else if (index == SRC_BLACK)
-    return SRC_BLACK_NAME;
```

Comment:
How are blocked requests recorded in dnsmasq's log now?

Reply:
They are currently logged only as a new query without a "result line". I thought about this as being a good realization at in keeping the dnsmasq changes minimal, however, I agree that having a result logging seems a good idea.

Reply:
The most recent commit adds back logging for blocked domains to `pihole.log`. Implementing it 1:1 to what we have now would require changes in dnsmasq's logging routine, however, I think my new way of logging is close enough (1 line addition):
```
Sep 24 11:32:11 dnsmasq[2081]: query[A] gravity-blocked.test.pi-hole.net from 127.0.0.1
Sep 24 11:32:11 dnsmasq[2081]: gravity blocked gravity-blocked.test.pi-hole.net is 0.0.0.0

Sep 24 11:32:13 dnsmasq[2081]: query[A] blacklist-blocked.test.pi-hole.net from 127.0.0.1
Sep 24 11:32:13 dnsmasq[2081]: exactly blacklisted blacklist-blocked.test.pi-hole.net is 0.0.0.0

Sep 24 11:32:17 dnsmasq[2081]: query[A] regex-blocked.test.pi-hole.net from 127.0.0.1
Sep 24 11:32:17 dnsmasq[2081]: regex blacklisted regex-blocked.test.pi-hole.net is 0.0.0.0
```